contracts/PublicResolver.sol

```diff
@@ -30,6 +32,7 @@ contract PublicResolver {
         bytes32 content;
         string name;
         PublicKey pubkey;
+        mapping (string => string) text;
```

Comment:
Can you update this to be consistent with the spacing of the rest of the entries?